doc/intl/FR_bzh/README.md

```diff
@@ -0,0 +1,92 @@
+<h1 align="center">
+<img src="https://static.tumblr.com/ppdj5y9/Ae9mxmxtp/300coin.png" alt="Dogecoin" width="300"/>
```

Comment:
Replace the Tumblr link with the logo in the repo `<img src="https://raw.githubusercontent.com/dogecoin/dogecoin/master/share/pixmaps/dogecoin256.svg" alt="Dogecoin" width="256"/>`